config/locales/mailers/en.yml

```diff
@@ -6,20 +6,20 @@ en:
         expiration:
           one: If you do not use this link to reset your password within %{count} day, it will expire, and you will have to request a new one.
           other: If you do not use this link to reset your password within %{count} days, it will expire, and you will have to request a new one.
-        intro: 'Someone has requested a password reset for your account. You can change your account password by following the link below and entering your new password:'
+        intro: "Someone has requested a password reset for your account. You can change your account password by following the link below and entering your new password:"
```

Comment:
The style guideline regarding `"` quotation marks being preferred does not apply to the locale yml files. The normalized files are expected to use `'` when needed.

You can see this because the i18n test for this file [fails](https://github.com/otwcode/otwarchive/actions/runs/8118300579/job/22192291997?pr=4760#step:12:1592). To have to show up as a failed test here on github, merge `master` into your branch so that you get [this bug fix](https://github.com/otwcode/otwarchive/commit/321fee426a60d87b5cd918fec93ec90a8c367116).

Reply:
(You can normalize the file again with `bundle exec i18n-tasks normalize -l en`, from https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#i18n-tasks.)

Reply:
I ran `bundle exec i18n-tasks normalize -l en # Normalize formatting of en.yml` in my docker container and it didn't appear to change the file (in the container, nor in my local codebase). 

I can make the changes manually, though. Just to be clear, all instances of `"` (double quotes) should be `'` (single quotes)?

Thank you

Reply:
Actually maybe that command fixed it because I just checked and most of the cases where `"` was used now it's `'`. So we may be good!